lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -1160,7 +1160,8 @@ class ADContext {
   /// Get or create an associated function index subset thunk from
   /// `actualIndices` to `desiredIndices` for the given associated function
   /// value and original function operand.
-  SILFunction *getOrCreateSubsetParametersThunkForLinearMap(
+  std::pair<SILFunction *, SubstitutionMap>
```

Comment:
Add a sentence in the doc comment to describe what the returned `SubstitutionMap` is and how it will be used.

Reply:
Done in https://github.com/apple/swift/pull/27604/commits/6f3156c3ad9cd6754ec5866f2d8a090c7b98e36f, thanks.